### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
<html>
<body>

Site | URLScan link
-- | --
synctoksrectify.com|https://user-images.githubusercontent.com/49607867/167346613-a551096a-0424-4277-b855-3267442a7418.png
syncmaintoken.com|https://user-images.githubusercontent.com/49607867/167346751-0e5de6e8-2fa1-4ffb-9cb8-d2658381ab6c.png
metaalliancedapps.com|https://user-images.githubusercontent.com/49607867/167346943-9055bf97-dd5c-4875-9684-25a0859da81d.png
networkrestoreauth.live|https://urlscan.io/result/c29b5456-f5ad-4bf8-9e80-a0b402588307/dom/
token-validate.com|https://user-images.githubusercontent.com/49607867/167349045-f1373476-0d48-4f77-acc7-d87303ed5fb3.png
tokensforensic.com|https://urlscan.io/result/d76f122c-c0e4-4e33-998d-4f0b4528f937/dom/
tokenpockat.icu|https://urlscan.io/result/b10dcb68-9a9f-4766-bad9-2c7dd2fbf7f4/

</body>
</html>

"dappsaccess-bots.net",
"dappsolutionfix.com",
"imtoken-3w.xyz",
"imtoken-gd.xyz",
"imtoken-h4.xyz",
"imtoken-h9.xyz",
"imtoken-p9.xyz",
"invest-token.art",
"metaalliancedapps.com",
"metamask-verify.rf.gd",
"metamask.io-toleuhfi.ru",
"metamaskapplog.com",
"metamaskkeyverif.com",
"mymetamaskwallet.ddns.net",
"swapdex.tokens-aword.com",
"syncmaintoken.com",
"synctoksrectify.com",
"synwalletconnect.site",
"token-conversion.com",
"token-validate.com",
"tokenapp.store",
"tokenpockat.icu",
"tokensforensic.com",
"validate-token.com",
"walletsdauthenticate.online",
"lnl-lawfirm.net",
"wapp.lnl-lawfirm.net",

### CODE REVIEW

**Review by dubstard — COMMENTED** [review]
remove duplicates
```
"dappsolutionfix.com",
"walletsdauthenticate.online",
```

**Review by dubstard — COMMENTED** [review]
add
		"autoappfix.com",
		"networkrestoreauth.live",
		"trasputmang.com",

File: all.json [hunk 1]
```diff
@@ -14,6 +14,34 @@
...
+		"imtoken-h9.xyz",
+		"imtoken-p9.xyz",
+		"invest-token.art",
+		"metaalliancedapps.com",
+		"metamask-verify.rf.gd",
+		"metamask.io-toleuhfi.ru",
+		"metamaskapplog.com",
+		"metamaskkeyverif.com",
+		"mymetamaskwallet.ddns.net",
+		"swapdex.tokens-aword.com",
```
Comment (jacogr) [inline all.json:33]: This is the only one that pops out to me - I see the "connect your metamask", but cannot determine if this is an actual platform on a funds stealer.

Comment (dubstard) [reply all.json:33]: `https://swapdex.tokens-aword.com/metamask/?nkbihfbeogaeaoehlefnkodbefgpgknn/home#restore-vault` is not a real metamask pop up but a fake one
even if you don't have the metamask plugin it imitates the place where it usually pops up and steals seed phrase

Comment (dubstard) [reply all.json:33]: image
this screenshot should suffice ...

Comment (jacogr) [reply all.json:33]: Brilliant, thanks. We need to have `tokens-aword.com` in the list based on the above.

**Review by dubstard — COMMENTED** [review]
all those are fake
```
4art.tokens-aword.com
air-nfts.tokens-aword.com
airnfts.tokens-aword.com
allme.tokens-aword.com
alyattes.tokens-aword.com
apecoin.tokens-aword.com
astroelon.tokens-aword.com
bingobet.tokens-aword.com
bitbook.tokens-aword.com
helenafinancial.tokens-aword.com
illuvium.tokens-aword.com
lanceria.tokens-aword.com
leagueofkingdoms.tokens-aword.com
lord.tokens-aword.com
luffy.tokens-aword.com
open-sea.tokens-aword.com
polygontechnology.tokens-aword.com
portion.tokens-aword.com
powerledger.tokens-aword.com
rarible.tokens-aword.com
refereum.tokens-aword.com
senspark.tokens-aword.com
swapdex.tokens-aword.com
tattoomoney.tokens-aword.com
thebifrost.tokens-aword.com
```

File: all.json (hunk 1 above)
Comment (dubstard) [reply all.json:33]: Yup yup, added the mail domain, but didn't bother with all the rest.

**Comment by dubstard** [comment]
h/t bunny and ecarlesi
✅Ready for review

**Review by jacogr — APPROVED** [review]

**Comment by dubstard** [comment]
thanks

**Pull request merged**